miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/permission/MiniAppPermission.kt

```diff
@@ -20,6 +20,7 @@ enum class MiniAppCustomPermissionType(val type: String) {
     PROFILE_PHOTO("rakuten.miniapp.user.PROFILE_PHOTO"),
     CONTACT_LIST("rakuten.miniapp.user.CONTACT_LIST"),
     ACCESS_TOKEN("rakuten.miniapp.user.ACCESS_TOKEN"),
+    SEND_MESSAGE("rakuten.miniapp.user.SEND_MESSAGE"),
```

Comment:
I set the value in JS SDK as `rakuten.miniapp.action.SEND_MESSAGE` so can we change this?

Reply:
Ah I see, I have followed iOS which has been merged already: 
https://github.com/rakutentech/ios-miniapp/blob/5ed44f98b7328eff4ba4294ca24c9f3b99e3b9a5/MiniApp/Classes/core/JavascriptBridge/MiniAppJavaScriptEnumerations.swift#L47 

I think, sending message is related to user permission, so we can use `user` for it 🤔

Reply:
Sending message is an action which is not related to object permission concept so I think `action` here is more reasonable than `user`.
@rleojoseph Can we change into `rakuten.miniapp.action.SEND_MESSAGE` for ios?

Reply:
@minh-rakuten I guess it makes sense. May be something like `rakuten.miniapp.user.action.SEND_MESSAGE` ? @corycaywood what do you think of this ?

Reply:
Yeah, I have updated using `rakuten.miniapp.user.action.SEND_MESSAGE`, Thanks for the discussion.